.github/workflows/csv-coverage-pr-comment.yml

```diff
@@ -0,0 +1,66 @@
+name: Comment on PR with framework coverage changes
+
+on:
+  workflow_run:
+    workflows: ["Check framework coverage changes"]
+    types:
+      - completed
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+    if: >
+      ${{ github.event.workflow_run.event == 'pull_request' &&
+      github.event.workflow_run.conclusion == 'success' }}
+
+    steps:
+    - name: Dump GitHub context
+      env:
+        GITHUB_CONTEXT: ${{ toJSON(github) }}
+      run: echo "$GITHUB_CONTEXT"
+    - name: Clone self (github/codeql) head
+      uses: actions/checkout@v2
+      with:
+        path: head
+    - name: Set up Python 3.8
+      uses: actions/setup-python@v2
+      with:
+        python-version: 3.8
+
+    # download artifacts from the PR job:
+    - name: Download artifact - HEAD
+      uses: dawidd6/action-download-artifact@v2.14.0
+      with:
+        workflow: csv-coverage-pr-artifacts.yml
+        run_id: ${{ github.event.workflow_run.id }}
+        name: csv-framework-coverage-merge
+        path: out_head
+
+    - name: Download artifact - BASE
+      uses: dawidd6/action-download-artifact@v2.14.0
+      with:
+        workflow: csv-coverage-pr-artifacts.yml
+        run_id: ${{ github.event.workflow_run.id }}
+        name: csv-framework-coverage-base
+        path: out_base
+
+    - name: Download artifact - PR
+      uses: dawidd6/action-download-artifact@v2.14.0
+      with:
+        workflow: csv-coverage-pr-artifacts.yml
+        run_id: ${{ github.event.workflow_run.id }}
+        name: pr
+        path: pr
+
+    - name: Check coverage files
+      run: |
+        PR=$(cat "pr/NR")
```

Comment:
You might not actually need this! `github.event.workflow_run.pull_requests[0].number` will give you this information.

Reply:
The `pull_requests` field is empty: https://github.com/dsp-testing/codeql-csv-coverage-pr-commenter/runs/2793922581?check_suite_focus=true

Reply:
Oh that's inconvenient... I wonder why. Then this is fine for now.